tests/routers/test_browser_security.py

```
@@ -0,0 +1,107 @@
+
+from app.models import MonitoredPath, OperationType, User
+from app.security import hash_password
+
+
+def test_path_traversal_viewer_forbidden(client: TestClient, db_session: Session):
+    """
+    Test that a viewer cannot access arbitrary paths like root /.
+    """
+    # 1. Create a viewer user
+    username = "viewer"
+    password = "password"
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to: "password"

(S105)